Client/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift

```diff
@@ -157,43 +154,58 @@ extension BrowserViewController: WKNavigationDelegate {
     }
 
     func webView(_ webView: WKWebView, decidePolicyFor navigationResponse: WKNavigationResponse, decisionHandler: @escaping (WKNavigationResponsePolicy) -> Void) {
+        let response = navigationResponse.response
+        let responseURL = response.url
+
         var request: URLRequest?
-        if let url = navigationResponse.response.url {
+        if let url = responseURL {
             request = pendingRequests.removeValue(forKey: url.absoluteString)
         }
 
         // We can only show this content in the web view if this URL is not pending
         // download via the context menu.
-        let canShowInWebView = navigationResponse.canShowMIMEType && (navigationResponse.response.url != pendingDownloadURL)
-        let forceDownload = navigationResponse.response.url == pendingDownloadURL
-
-        let openInHelper = OpenIn.helperForRequest(request, response: navigationResponse.response, canShowInWebView: canShowInWebView, forceDownload: forceDownload, browserViewController: self)
-        guard var helperForURL = openInHelper, helperForURL is DownloadHelper else {
-            if navigationResponse.canShowMIMEType {
-                addViewForOpenInHelper(openInHelper)
-                decisionHandler(.allow)
-                return
-            }
+        let canShowInWebView = navigationResponse.canShowMIMEType && (responseURL != pendingDownloadURL)
+        let forceDownload = responseURL == pendingDownloadURL
 
-            let error = NSError(domain: ErrorPageHelper.MozDomain, code: Int(ErrorPageHelper.MozErrorDownloadsNotEnabled), userInfo: [NSLocalizedDescriptionKey: Strings.UnableToDownloadError])
-            ErrorPageHelper().showPage(error, forUrl: navigationResponse.response.url!, inWebView: webView)
```

Comment:
This is removed and not added back in. Is this alright?

Reply:
Yes, I believe so. We shouldn't hit a case anymore where we can't show content in the webview without prompting to download.